xdebug/README.md

```diff
@@ -0,0 +1,136 @@
+# Set up XDebug
```

Comment:
Please add some short intro, just a sentence or two about what Xdebug is like on other pages.